lib/iris/common/metadata.py

```diff
@@ -37,6 +37,7 @@
     "CoordMetadata",
     "CubeMetadata",
     "DimCoordMetadata",
+    "ConnectivityMetadata",
```

Comment:
Alphabetical? i.e., slot in after `CellMeasureMetadata`

Reply:
c2ac0fc05949f07cd3483e802994708aa427da22